.devcontainer/compose.yaml

```diff
@@ -1,13 +1,16 @@
-name: "rails8_devcontainer_enhancements"
+name: "rails8_devcontainer_enhancements_devcontainer"
```

Comment:
Does not work 100% as expected, see https://github.com/microsoft/vscode-remote-release/issues/2485 and https://github.com/devcontainers/cli/issues/306. But, at least I can reuse the same images and volumes.